database/migrations/2022_06_12_075709_add_token_to_user_table.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use App\Models\Configs;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class AddTokenToUserTable extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 *
+	 * @throws Exception
+	 */
+	public function up(): void
+	{
+		Configs::where('key', '=', 'api_key')->delete();
+
+		Schema::table('users', function (Blueprint $table) {
+			$table->char('token', 100)->after('email')->default('');
+		});
+	}
+
+	/**
+	 * Reverse the migrations.
+	 *
+	 * @return void
+	 */
+	public function down(): void
+	{
+		DB::table('configs')->insert([
+			[
+				'key' => 'api_key',
+				'value' => '',
+				'confidentiality' => 3,
+				'cat' => 'Admin',
+			],
+		]);
+		Schema::disableForeignKeyConstraints();
+		Schema::table('users', function (Blueprint $table) {
+			$table->dropColumn('token');
+		});
```

Comment:
The ongoing problem. SQLite does not support dropping columns yet. Dropping a column is mimic by re-creating a table without that column. But this throws away all foreign key constraints.

For the time being, we must abstain from dropping columns. But luckily this is only in the downwards path, not the upwards path. Otherwise we had to add it to the list in https://github.com/LycheeOrg/Lychee/issues/1321.